### DISCUSSION TITLE
mod_conversejs with ejabberd 24.10

### DISCUSSION THREAD

**Discussion by radatus** [processone/ejabberd#4317] [General]
I tried to install the converse js module for ejabberd but did not find it in the list of modules as stated in the instructions https://docs.ejabberd.im/admin/configuration/modules/#mod-conversejs

I installed ejabberd 24.10. Set it up. It works fine. But how to set up converse.js I don't understand.

How to make it work converse.js in ejabberd 24.10 ?

**Comment by badlop** [comment]
I tried ejabberd 24.10 binary installer and works correctly.

1. Installed `ejabberd-24.10-1-linux-x64.run`
2. The module binary file is available in `/home/badlop/opt/ejabberd-24.10/lib/ejabberd-24.10.0/ebin/mod_conversejs.beam`
3. Now I configure the module in ejabberd as specified in the first example in https://docs.ejabberd.im/admin/configuration/modules/#mod_conversejs
4. Restart ejabberd
5. Open web browser http://localhost:5280/conversejs

In that simple example setup ejabberd serves the public Conversejs client.

For a production server, it is preferable to download Conversejs client to your server machine and let ejabberd serve that local copy. There are examples in the documentation and in https://www.process-one.net/blog/ejabberd-21-12/

**Comment by radatus** [comment]
Great! Thanks, I'll try.
